compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -73,19 +96,33 @@ bool resolve_with_BroadcastTo(luci::CircleCustom *addv2)
   if (broadcastTo_idx == -1)
     return false;
 
-  auto input = loco::must_cast<const luci::CircleCustomOut *>(addv2->inputs(broadcastTo_idx));
-  auto broadcastTo = loco::must_cast<luci::CircleCustom *>(input->input());
+  auto broadcastTo = dynamic_cast<luci::CircleBroadcastTo *>(addv2->inputs(broadcastTo_idx));
 
   auto name = addv2->name();
   assert(name.length() > 0);
 
   auto add = addv2->graph()->nodes()->create<luci::CircleAdd>();
   add->fusedActivationFunction(luci::FusedActFunc::NONE);
   add->x(addv2->inputs(1 - broadcastTo_idx));
-  add->y(broadcastTo->inputs(0));
+
+  // BroadcastTo(Builtin)
```

Comment:
Original logic does two things sequentially.

1. find where the broadast_to operator is;it returns false if there is no such op.
2. create a new add operator.

But, this PR is creating a add operator first, which could become obsolete if there is no op. So, getting the op first will be efficient.